worlds/mlss/Items.py

```diff
@@ -184,6 +185,7 @@ class MLSSItem(Item):
     "Hoo Bean": 100,
     "Chuckle Bean": 200,
     "Hammers": 3,
+    "Victory": 0
```

Comment:
You'd want a comma here for consistency
```suggestion
    "Victory": 0,
```